src/components/getting_started/index.js

```diff
@@ -1,10 +1,10 @@
 import { Breadcrumb } from 'react-breadcrumbs';
 import { Redirect, Route, Switch } from 'react-router-dom';
 import DocumentTitle from 'react-document-title';
-import Page0_Overview from './0_overview.js';
-import Page1_ConfigureKubeCTL from './1_configure_kubectl.js';
-import Page2_SimpleExample from './2_simple_example.js';
-import Page3_NextSteps from './3_next_steps.js';
+import Page0_Overview from './0_overview';
```

Comment:
This doesn't really match our naming rules (because of the numbers and underscores)

Reply:
yeah I only changed the folder, not the components in here.

The PR is originally just for app catalog components, but noticed the hyphen in `getting-started` folder and decided to fix that quickly too.

I'll make other PR's for other components. But actually have been thinking of saving that as a starting task for @ubergesundheit so he can get a bit familiar with the layout of Happa's files.

Reply:
The [file name guide](https://github.com/giantswarm/fmt/blob/master/javascript/general_style_guide.md#naming) does not mention numbers being not allowed or did I miss something in the style document?

Which one of those should be the new file name? I think `Overview.js` is suited best.

- `PageZeroOverview.js`
- `Overview.js`
- `Page0Overview.js`
- `Page0.js`

Reply:
`Overview.js` and `NextSteps.js` is fine, I guess we don't need the page numbers in the component file names.